### ISSUE TITLE
Ember.js Tutorial

### ISSUE THREAD

**Issue by payne-chris-r** [ga-wdi-boston/ember-study#83]
So, I just ran through the emberjs tutorial, and I'm actually pretty impressed. Since we're switching to `default` over `pods` structure, I think we can have them drill through this at the start and it allows us to be much more succinct in the early `ember` talks.
Here is the tutorial I'm referring to: https://guides.emberjs.com/v2.11.0/tutorial/ember-cli/

As far as I can tell, only the bolded ones would be required (R), italics are helpful (H), plain text can be ignored or are `Nice to Have`s (NTH):

_TUTORIAL:_
**Creating Your App** (R) -> `ember-study`
Setting Up Tests <- We don't cover testing, maybe this is a (NTH)
**Routes and Templates** (R) -> `ember-routing-study`
**The Model Hook** (R) -> `ember-routing-study`
Installing Addons (NTH)
_Building a Simple Component_ (NTH)
_Creating a Handlebars Helper_ (NTH)
**Using Ember Data** (R) -> `ember-routing-study` || `ember-resources-study`
_Building a Complex Component_ (NTH) <- f*ck this is cool.
Services and Utilities (NTH)
Adding Nested Routes (R)  -> `ember-routing-study`
Deploying (NTH)

NOTE: In almost all the sections there are pieces on testing that we can tell them to ignore unless they're interested in implementing it themselves/would like an added challenge.

**Comment by jrhorn424** [comment]
> Building a Simple Component (NTH)
> Building a Complex Component (NTH) <- f*ck this is cool.

I vote required for the first and mention the second (additional resource, bonus, challenge, whatev) in the study `ember-compononents-study`.

> Using Ember Data (R) -> ember-routing-study || ember-resources-study

`ember-resources-study`

> Services and Utilities (NTH)

Required, but in `ember-auth-study`.

> Deploying (NTH)

Already covered in `ember-deployment-guide`. The notes from this guide relating to the location parameter should definitely be an objective of the `ember-routing-study`.

> Adding Nested Routes (R) -> ember-routing-study

I dunno. Nested routes have been an antipattern for many projects. I know @gaand has a bit more experience with them, so you and I should discuss this and then one of us can talk to @gaand about it when he can afford to chat.

(anything not mentioned in this reply, but in @payne-chris-r's previous comment, I agree with)

**Comment by MicFin** [comment]
Addressed for this repo but needs to be left open and split up into others per https://github.com/ga-wdi-boston/ember-study/pull/89

**Comment by jrhorn424** [comment]
> Addressed for this repo but needs to be left open and split up into others per #89

I'd like a second or third pair of eyes for confirmation.

Open split issues where appropriate, cross-reference them here. Once there is agreement all are accounted for, this can be closed. Not until then.
